docs/api_docs/python.rst

```diff
@@ -0,0 +1,170 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _getting_started_python_api:
+
+******************************
+Getting Started with Python
+******************************
+
+Installation
+==================
+.. note::
+    We heavily recommend to manage your projects with some sort of virtual environment, for example Pipenv. For instructions to install Pipenv for Mac or Windows click `here <https://medium.com/@mahmudahsan/how-to-use-python-pipenv-in-mac-and-windows-1c6dc87b403e>`_. For installation instructions for Linux click `here <https://github.com/pypa/pipenv>`_.
+
+Installing with Pipenv
+-----------------------------
+In order to install the Vertiq python API into your pipenv virtual environment, simply run ``pipenv install iqmotion`` within your virtual environment.
+
+Installing with Pip
+-----------------------------
+In order to install the Vertiq python API via pip, simply run ``pip install iqmotion``.
+
+Python API Basics
+=========================
+Hardware Setup
+----------------
+Information about required hardware for API communication can be found in the :ref:`Control Center documentation <connection_guide>`. The requirements for API communication 
+are the same as those for the IQ Control Center.
+
+Opening a Serial Connection
+--------------------------------
+In order to connect with your USB-to-UART bridge, the Python API provides the ``SerialCommunicator`` class.
+
+The ``SerialCommunicator`` class is initialized with a serial port name and a baud rate. The name of your serial port is dependent on your operating system. 
+For example, in the hardware example provided in the Control Center documentation linked above, the serial port name is “COM3,” and in Linux, it may be "/dev/ttyUSB0."
+
+In order to create a new ``SerialCommunicator`` object on a Windows operating system where your module is set to 115200 baud, and your FTDI reports on COM9, you would do the following:
+
+.. code-block:: python
+    
+    import iqmotion as iq
+    com = iq.SerialCommunicator("COM9", baudrate=115200)
+
+In order to communicate with a Vertiq module through the Python API, you must create a ``SerialCommunicator`` object.
+
+Creating a Module Object
+-----------------------------
+Vertiq's module objects are preconfigured to communicate with all accessible IQUART clients for your module's firmware. All module objects are created with the same 
+initialization.
+
+1. A serial communicator object
+2. Optionally, a module ID (``module_idn``)
+
+  * This represents the module ID of the device you would like to communicate with. If your module has its module ID configured to 26, you must set ``module_idn`` to 26 in object instantiation. 
+    By default, all modules have an ID of 0.
```

Comment:
When you say "By default, all modules have an ID of 0", I'm a little unsure if you mean the module objects in the API have an ID of 0 by default, or if the actual module firmware has an ID of 0 by default. Obviously I know the answer is both, but here it may be worth clarifying a bit more, e.g. "If no module ID is specified, the module object is created with an ID of 0".